src/public/modules/forms/admin/controllers/collaborator-modal.client.controller.js

```diff
@@ -136,6 +142,33 @@ function CollaboratorModalController(
     }
   }
 
+  /**
+   * Removes the current user from the collaborator list and bounces the user back to the forms select page
+   */
+  $scope.removeSelfFromCollab = () => {
+    return $q
+      .when(UpdateFormService.removeSelfFromCollaborators($scope.myform._id))
+      .then(() => {
+        // redirects the user back to the home page now that they have lost access to this form
+        $window.location.href = $window.location.origin
```

Comment:
@mantariksh @liangyuanruo I am pretty sure this is not the best way to redirect the user to the home page or the front page but I am not sure what the canonical way is.

Reply:
I think redirecting the user back to the dashboard containing their list of forms would be more intuitive. I tested this and it seems to work (you'll need to replace the `$window` service with `$state`):
```
$scope.toggleRemoveSelfFromCollabModal()
$scope.closeModal()
$state.go('listForms')
```